api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/handler/AbstractHandler.java

```diff
@@ -1,11 +1,234 @@
 package org.openmrs.module.fhir.api.diagnosticreport.handler;
 
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.hl7.fhir.dstu3.model.*;
```

Comment:
I believe you should unwrap this wildcard.

Reply:
I was going to unwrap wildcards from all classes in this repository in my next pull request

Reply:
Ok